### DISCUSSION TITLE
false negative & false positive for 8k Chinese phone record

### DISCUSSION THREAD

**Discussion by songtaoshi** [snakers4/silero-vad#96] [Q&A]
Hi @snakers4 , really impressive project !  
I am using this awesome vad in my project, but I find some false negative & false positive examples in my experiments, it's really hard to find a good parameters(trig, neg_trig, min_speech, min_silence).   
My dataset is Chinese Phone records of 8k sample rate, I really believe this project can work, but can you give me some help about what I can do just using existing models.   
best regards

**Comment by snakers4** [comment]
Hi,

- Which model and interface are you using?
- Can you post a chart that is generated with `visualize_probs=True`?
